src/main/resources/application.conf

```diff
@@ -34,7 +34,8 @@ spray.can {
 
 
 backend {
-  backend = "local" // Either "JES" or "local", case insensitive
+  // Either "jes", "local", or "sge" (case insensitive)
+  backend = "sge"
```

Comment:
Just double checking this is just for your noodling around. It should stay as "local" in github as the most generic default.